packages/design-system/src/DatePicker/styles.css

```diff
@@ -0,0 +1,165 @@
+.react-datepicker.ads-v2-datepicker__calender {
```

Comment:
:( 

Can you leave a link to where the documentation for these classes are at the top of this file?

Reply:
Also comments above each bit will be very nice.